### DISCUSSION TITLE
Error after make:migration

### DISCUSSION THREAD

**Discussion by guiwestrup** [adalessa/laravel.nvim#53] [Q&A] [answered]
After doing a migration, it gives this message:

<img width="1352" alt="Captura de Tela 2023-11-02 às 14 33 08" src="https://github.com/adalessa/laravel.nvim/assets/23106111/2cb43d15-7593-428a-a501-a4bc38cd1eea">

**Comment by adalessa** [comment, marked as answer]
Thanks for letting me know. I use `fd` to find the new file in the system using regular expression since the migration name is generated there is no easy way to get it. 
I recommend install it https://github.com/sharkdp/fd. 
I will have to update the docs and add a wrapper and a notification if not to let the user know

**Reply by guiwestrup** [reply]
Nice, worked perfectly.

**Comment by adalessa** [comment]
#54 created
